corehq/apps/data_interfaces/models.py

```diff
@@ -888,7 +888,7 @@ def when_case_matches(self, case, rule):
 
         if close_case or properties:
             result = update_case(case.domain, case.case_id, case_properties=properties, close=close_case,
-                xmlns=AUTO_UPDATE_XMLNS)
+                                 xmlns=AUTO_UPDATE_XMLNS, max_wait=15)
```

Comment:
Can you talk about the choice to apply this just to auto case updates and not, say, case imports? If you did want to throttle all system forms like this, you could do that by just changing the default `max_wait` for `submit_form_locally` here https://github.com/dimagi/commcare-hq/blob/7e6cea347aa37412b541dd199be22a4785035def/corehq/apps/receiverwrapper/util.py#L42-L43 from 0.1 (100 ms) to 15. You're probably not wrong, just curious to hear the thinking.

Reply:
I was trying to limit the affected areas to only those I knew. I definitely think it is worth going through all the places that use `submit_form_locally` and seeing if it is safe to apply this change to all of them, but I wasn't confident enough yet and wanted to make sure this got out while I was checking